doc/admin-guide/plugins/header_rewrite.en.rst

```diff
@@ -775,6 +775,14 @@ changing the remapped destination, ``<part>`` should be used to indicate the
 component that is being modified (see `URL Parts`_). Currently the only valid
 parts for rm-destination are QUERY, PATH, and PORT.
 
+run-plugin
+~~~~~~~~~~~~~~
+::
+
+  run-plugin <plugin-name>.so "<plugin-argument> ..."
+
+This allows to run an existing remap plugin, conditionally, from within a
+header rewrite rule.
```

Comment:
Can this run on any hook other than the remap hook?

Reply:
No it can only run in a remap hook, but the plugin it runs can setup other hooks.